pkg/resolver/search.go

```diff
@@ -291,48 +309,53 @@ func WhereClauseFilter(ctx context.Context, input *model.SearchInput,
 
 	if input.Filters != nil {
 		for _, filter := range input.Filters {
-			if len(filter.Values) > 0 {
-				values := pointerToStringArray(filter.Values)
-
-				if len(propTypeMap) > 0 {
-					if dataTypeInMap, ok := propTypeMap[filter.Property]; !ok { // check if value exists
-						klog.Warningf("Property type for [%s] doesn't exist in cache. Refreshing property type cache",
-							filter.Property)
-						propTypeMapNew, err := getPropertyType(ctx, true) // Refresh the property type cache.
-						if err != nil {
-							klog.Warningf("Error creating property type map with err: [%s] ", err)
-						}
-						propTypeMap = propTypeMapNew
-						break
-
-					} else {
-						klog.V(5).Infof("For filter prop: %s, datatype is :%s\n", filter.Property, dataTypeInMap)
-
-						// if property mactches then call decode function:
-						values, dataTypeFromMap = decodePropertyTypes(values, dataTypeInMap)
-						// Check if value is a number or date and get the cleaned up value
-						opDateValueMap = getOperatorAndNumDateFilter(filter.Property, values, dataTypeFromMap)
-					}
-				} else {
-					klog.Error("Error with property type list is empty.")
-					values = decodePropertyTypesNoPropMap(values, filter)
-					// Check if value is a number or date and get the cleaned up value
-					opDateValueMap = getOperatorAndNumDateFilter(filter.Property, values, nil)
-
-				}
-				//Sort map according to keys - This is for the ease/stability of tests when there are multiple operators
-				keys := getKeys(opDateValueMap)
-				var operatorWhereDs []exp.Expression //store all the clauses for this filter together
-				for _, operator := range keys {
-					operatorWhereDs = append(operatorWhereDs,
-						getWhereClauseExpression(filter.Property, operator, opDateValueMap[operator], dataTypeFromMap)...)
+			if len(filter.Values) == 0 {
+				klog.Warningf("Ignoring filter [%s] because it has no values", filter.Property)
+				continue
+			}
```

Comment:
Maybe this condition should return an error,  but this works for the purpose of this PR.

Reply:
Actually, graphql itself will return an error here as the values are mandatory. And on the UI, the `Run Search` button is inactive without it.